tiingo/api.py

```diff
@@ -205,7 +246,8 @@ def get_ticker_price(self, ticker,
             return response.content.decode("utf-8")
 
     def get_dataframe(self, tickers,
-                      startDate=None, endDate=None, metric_name=None, frequency='daily'):
+                      startDate=None, endDate=None, metric_name=None,
+                      frequency='daily', fmt='json'):
```

Comment:
Happy to see that this preserves usage signature for anyone already using this method 👍